src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-modification-form.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Grid from '@mui/material/Grid';
+import {
+    CHARACTERISTICS_CHOICE,
+    CHARACTERISTICS_CHOICES,
+    Q_AT_NOMINAL_V,
+    SHUNT_COMPENSATOR_TYPE,
+    SHUNT_COMPENSATOR_TYPES,
+    SUSCEPTANCE_PER_SECTION,
+} from 'components/utils/field-constants';
+import { Box } from '@mui/material';
+import { useWatch } from 'react-hook-form';
+import { FloatInput } from '@gridsuite/commons-ui';
```

Comment:
can you merge the imports from '@gridsuite/commons-ui' please

Reply:
Done